db-client/src/main/java/com/splicemachine/db/client/net/NetConnectionReply.java

```diff
@@ -714,6 +715,14 @@ private void parseACCRDBRM(NetConnection netConnection) throws DisconnectExcepti
                 peekCP = peekCodePoint();
             }
 
+            if (peekCP == CodePoint.RDBINTTKN) {
+                // carries information to abort running command
+                foundInPass = true;
+                rdbinttknReceived = checkAndGetReceivedFlag(rdbinttknReceived);
+                rdbinttkn = parseRDBINTTKN(false);
```

Comment:
Is there any relation between `foundInPass = true` and `parseRDBINTTKN(false)`? If so, what about `parseRDBINTTKN(!foundInPass)` to make it more obvious?